examples/huggingface-transformers/README.md

```diff
@@ -0,0 +1,137 @@
+
+# SparseZoo model stub or path to ONNX file
+onnx_filepath='zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned-moderate'
+
+num_cores=None  # uses all available CPU cores by default
+
+# Get DeepSparse question-answering pipeline
+
+qa_pipeline = pipeline(
+    task="question-answering",
+    model_path=onnx_filepath,
+    num_cores=num_cores,
+)
+
+# inference
+
+my_name = qa_pipeline(question="What's my name?", context="My name is Snorlax")
+```
+The pipeline can also infer a default sparse model to run on the system
+
+```python
+from deepsparse.transformers import pipeline
+qa_pipeline = pipeline('question-answering')
+my_name = qa_pipeline(question="What's my name?", context="My name is Snorlax")
+```
+
+## Benchmarking Example
+`benchmark.py` is a script for benchmarking sparsified Hugging Face Transformers
+performance with DeepSparse.  For a full list of options run `python 
+benchmark.py -h`.
+
+To run a benchmark using the DeepSparse Engine with a pruned BERT model that uses all available CPU cores and batch size 1, run:
+```bash
+python benchmark.py \
+    zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned-moderate \
+    --batch-size 1
+```
+
+
+
+## Example [Transformers](https://github.com/huggingface/transformers) DeepSparse Flask Deployment
+
+To illustrate how the DeepSparse Engine can be used with Hugging Face 
+Transformers, this directory contains a sample model server and client. 
+
+The server uses Flask to create an app with the DeepSparse Engine hosting a
+compiled Hugging Face Transformer model.
+The client can make requests into the server returning inference results for 
+given inputs.
+
+### Server
+
+First, start up the host `server.py` with your model of choice, SparseZoo stubs are
+also supported.
+
+Example command:
+```bash
+python server.py \
+    zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned-moderate
+```
+
+You can leave that running as a detached process or in a spare terminal.
+
+This starts a Flask app with the DeepSparse Engine as the inference backend, accessible at `http://0.0.0.0:5543` by default.
+
+The app exposes HTTP endpoints at:
+- `/info` to get information about the compiled model
+- `/predict` to send inputs to the model and receive a response.
+    The number of inputs should match the compiled model's batch size.
+
+For a full list of options, run `python server.py -h`.
+
+Currently, the server uses DeepSparse-HuggingFace pipeline integration 
+for end-to-end prediction.  
+
+### Client
+
+`client.py` provides a callable `PipelineClient` object to make requests to the 
+server easy.
+The file is self-documented.  See example usage below:
+
+```python
+from client import PipelineClient
+remote_model = PipelineClient(address='0.0.0.0', port='5543')
+model_outputs = remote_model(question="What's my name?", context="My name is Snorlax")
+```
+
+### SparseZoo Stubs
+
+
+| Model Name     |      Stub      |
```

Comment:
let's add an extra column for a brief model description

Reply:
Done!